src/components/auction/AuctionDetails/index.tsx

```diff
@@ -431,7 +431,7 @@ const AuctionDetails = (props: Props) => {
         tooltip: 'Address of the contract managing the allow-list for participation',
         url: `https://etherscan.io/address/${auctionDetails?.allowListManager}`,
         value: `${
-          auctionDetails && auctionDetails.allowListManager == ''
+          auctionDetails && auctionDetails.allowListManager !== ''
             ? auctionDetails.allowListManager.substr(0, 6).concat('...')
```

Comment:
Alex, what do you think if we use of the methods we already have for this? ([shorenAddress](https://github.com/gnosis/ido-ux/blob/showAddresses/src/utils/index.ts#L73) or [truncateStringInTheMiddle](https://github.com/gnosis/ido-ux/blob/showAddresses/src/utils/tools.ts#L3))
with `shortenAddress(address, 3)`  we would show 3 characters at both ends like **0x7C8...AE9**